addons/resource.language.en_gb/resources/strings.po

```diff
@@ -12863,7 +12863,7 @@ msgstr ""
 
 #: system/settings/settings.xml
 msgctxt "#20433"
-msgid "Extract thumbnails and video information"
+msgid "Extract video information from files"
```

Comment:
It is still used: https://github.com/popcornmix/xbmc/blob/838a9224e48303e4a623cc7c036525fc4d2f44b7/system/settings/settings.xml#L967